hack/utils/installdefinition.sh

```diff
@@ -0,0 +1,24 @@
+#! /bin/bash
+DEFPATH="../../charts/vela-core/templates/defwithtemplate"
+function scan_def(){
+res=`kubectl get namespace -A | grep vela-system`
+if [ -n "$res" ];then
+  echo 'vela-system namespace exist'
+else
+  echo 'vela-system namespace do not exist'
+  echo 'creating vela-system namespace ... '
+  kubectl create namespace vela-system
+fi
+echo "applying definitions ..."
+cd $DEFPATH
+# shellcheck disable=SC2045
+for file in `ls .`
+  do
+    echo "Info: changing "$DEFPATH"/"$file
+    sed -i '' "s#{{.Values.systemDefinitionNamespace}}#vela-system#g" $file
+    kubectl apply -f $file
+    sed -i '' "s#vela-system#{{.Values.systemDefinitionNamespace}}#g" $file
```

Comment:
what if the file contains many `vela-system`?

Reply:
I suggest use `sed -i.bak`

1. different system has same command. both linux and macos
2. you just need to delete the backup file after apply

Reply:
fix